packages/kit-bg/src/states/jotai/atoms/prime.ts

```
@@ -6,6 +6,7 @@ import type {
 
 import { EAtomNames } from '../atomNames';
 import { globalAtom } from '../utils';
+import { IAccountDeriveTypes } from '../../../vaults/types';
```

ESLint

[error] All imports in the declaration are only used as types. Use `import type`.

(@typescript-eslint/consistent-type-imports)